kraken-build/pyproject.toml

```diff
@@ -18,7 +18,7 @@ packages = [
   {include = "kraken/std", from = "src"},
 ]
 readme = "README.md"
-version = "0.32.4"
+version = "0.32.4.post0.dev2+dirty"
```

Comment:
Undo those please :)

You don't need to run the `build` task for local dev, `fmt lint test` is enough

Reply:
Sure! Honestly I don't know what I'm doing, I saw those in the diff before making the commit and I already figured that they don't belong 😛

Reply:
Done — just pushed a commit. I did a `git rebase`, `git reset --soft HEAD~1` and `git restore`'d every file which contains a dirty version (some `pyproject.toml` files, some `__init__.py` files). Hope that is right!